### ISSUE TITLE
Color indicator for crits in role results

### ISSUE THREAD

**Issue by mdiviesti** [foundryvtt/dnd5e#4089]
In previous versions, rolls of nat 1 and nat 20 used to have an indicator in the role results in chat. 
This is no longer happening in v331. 
One must expand to view the rolls to see the dice color. 

Additionally, it would be awesome if whenever there was a nat 1 or nat 20 on the roll that the die icon itself would appear next to the result for easier viewing. 

I have color blind players who can't tell the difference between red and green.

**Comment by Rabulias** [comment]
Have you tested this with no modules? Are these manual rolls? Attack rolls from the sheet by default highlight as green or red on success (depending on your settings for Challenge Visibility and Attack Result Visibility). Even you have those set to **Hide all**, natural 1s and 20s on Attack Rolls show as red and green. Natural 1s will have two Xs to the left of the result, and natural 20s will have two check marks.
![Crits](https://github.com/user-attachments/assets/186852c8-075b-42a4-bc1b-bf1629f3e7b5)

**Comment by Rabulias** [comment]
Also, remember that by the standard rules, natural 1s and natural 20s only affect Attack rolls -- natural rolls have no effect on Skill Checks, Saving Throws, and other Ability Checks.

**Comment by mdiviesti** [comment]
I'm digging deeper into this issue and here's what I'm seeing: 
First of note: our world began during V10 of Foundry (don't remember which version of the 5e game module it was). 
We upgraded to 11 after it came out. 
Now I'm testing an upgrade to 12 (copied v11 foundrydata folder and did the normal backup-migration path) and updated to the latest dnd module.

When I first reported this "issue", I was going through older rolls and noticing that none of them were highlighted for any crits (attack, or challenge) and yes, we do all of our rolls from sheets for attacks and ability checks. 

To test what was happening here, I turned off all modules. Still all previous rolls had no indicators, even after changing Challenge Visibility to "Show all" and "Attack Result Visibility" to "Show results & target ACs". 

So to test, I made some rolls and yes, Attacks for new rolls are showing up properly (though historical rolls have no indicator). 
Skill Checks, however still are not showing crits for new rolls. 

Yes I know that RAW doesn't use crits on skill checks, but lots of people use them to flavor the result even if they don't use them to make an automatic pass/fail.

So to clarify, at present this bug should be narrowed down a little. 
The indicator is only not showing on historical rolls migrated during v11->v12 upgrade and for skill checks.
